app/layout.tsx

```diff
@@ -13,8 +13,8 @@ const geistMono = Geist_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "PawRide - Uber for Dogs",
+  description: "Premium transportation service for your furry friends",
```

Comment:
Yasss queen 👑! Line 17 is snacc 🍖. Making it hella clear with 'Premium transportation service for your furry friends'. Talk about a glow up! 😍✨